File: python/minimum-number-of-steps-to-make-two-strings-anagram-ii.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        char_count = defaultdict(lambda: 0)
        for c in s:
            char_count[c] += 1
        for c in t:
            char_count[c] -= 1
        steps = 0
        for c in char_count:
            steps += abs(char_count[c])
        return steps

File: go/minimum-number-of-steps-to-make-two-strings-anagram-ii.go
func minSteps(s string, t string) int {
	charCount := make([]int, 26)
	for i := 0; i < len(s); i++ {
		c := int(s[i] - 'a')
		charCount[c] += 1
	}
	for i := 0; i < len(t); i++ {
		c := int(t[i] - 'a')
		charCount[c] -= 1
	}
	steps := 0
	for _, v := range charCount {
		if v < 0 {
			v = -v
		}
		steps += v
	}
	return steps
}
